armi/bookkeeping/report/reportingUtils.py

```diff
@@ -188,6 +195,8 @@ def _writeMachineInformation():
                     tablefmt="armi",
                 )
             )
+            runLog.header("=========== System Information ===========")
+            runLog.info(sysInfo)
```

Comment:
Perhaps we can wrap these two lines in a check?

```python
if sysInfo:
    runLog.header("=========== System Information ===========")
    runLog.info(sysInfo)
``

Reply:
done!